sample/ground_segmentation/scenario_annotated_rosbag.yaml

```
@@ -0,0 +1,20 @@
+ScenarioFormatVersion: 3.0.0
+ScenarioName: ground_segmentation-sample
+ScenarioDescription: ground_segmentation-sample
+SensorModel: aip_xx1
+VehicleModel: jpntaxi
```

could you test with pilot-auto?
```suggestion
VehicleModel: lexus
```